File: Strings/344-reverse-string/344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        n=len(s) #len of n
        start=0 #start 
        end=n-1 #end
        while(start<=end):
            s[start],s[end]=s[end],s[start] #swap last with first and so on 
            start+=1
            end-=1
        
            
                
                
        

File: 344-reverse-string/344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int l=0;
        int r=s.size()-1;
        char temp=0;
        while(l<r)
        {
            temp=s[l];
            s[l]=s[r];
            s[r]=temp;
            l++;
            r--;
        }
    }
};